core/src/main/java/kafka/log/remote/RemoteLogManager.java

```diff
@@ -153,6 +156,8 @@ public class RemoteLogManager implements Closeable {
 
     private final RemoteLogMetadataManager remoteLogMetadataManager;
 
+    private final ReentrantLock copyQuotaManagerLock = new ReentrantLock(true);
```

Comment:
Why `fairness` is turned on to true?

Reply:
We want the threads that were blocked first to be unblocked first. This is to prevent the starvation of some threads.